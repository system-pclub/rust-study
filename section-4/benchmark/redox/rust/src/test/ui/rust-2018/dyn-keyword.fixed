// edition:2015
// run-rustfix

#![allow(unused_variables)]
#![deny(keyword_idents)]

fn main() {
    let r#dyn = (); //~ ERROR dyn
    //~^ WARN hard error in the 2018 edition
}
