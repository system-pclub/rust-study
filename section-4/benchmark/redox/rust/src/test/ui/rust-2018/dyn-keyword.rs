// edition:2015
// run-rustfix

#![allow(unused_variables)]
#![deny(keyword_idents)]

fn main() {
    let dyn = (); //~ ERROR dyn
    //~^ WARN hard error in the 2018 edition
}
