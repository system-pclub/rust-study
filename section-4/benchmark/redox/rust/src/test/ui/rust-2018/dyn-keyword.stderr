error: `dyn` is a keyword in the 2018 edition
  --> $DIR/dyn-keyword.rs:8:9
   |
LL |     let dyn = ();
   |         ^^^ help: you can use a raw identifier to stay compatible: `r#dyn`
   |
note: lint level defined here
  --> $DIR/dyn-keyword.rs:5:9
   |
LL | #![deny(keyword_idents)]
   |         ^^^^^^^^^^^^^^
   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: aborting due to previous error

